modules/sor/sor.service.ts

```diff
@@ -17,6 +17,9 @@ import { AllNetworkConfigsKeyedOnChain } from '../network/network-config';
 
 export class SorService {
     async getSorSwapPaths(args: QuerySorGetSwapPathsArgs): Promise<GqlSorGetSwapPaths> {
+        if(!!args.useVaultVersion && args.useVaultVersion!==2 && args.useVaultVersion!==3){
```

Comment:
Whats wrong with your prettier? This should be formatted

Reply:
Actually, this check should be different. If there is no vaultVersion specified, it should find routes on both versions and only return the better one.